src/LandIce/problems/LandIce_StokesFOBase.hpp

```diff
@@ -1463,16 +1457,20 @@ void StokesFOBase::constructSMBEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>
     // only some of the sub-sidesets of 'basalSideName'. The layout of all the side fields is the
     // same, so we need to differentiate them by name (just like we do for the basis functions already).
 
-    std::string velocity_side_name = velocity_name + "_" + basalSideName;
-    std::string ice_thickness_side_name = ice_thickness_name + "_" + basalSideName;
-    std::string ice_thickness_side_name_planar = ice_thickness_name + "_" + basalSideNamePlanar;
-    std::string surface_height_side_name = surface_height_name + "_" + basalSideName;
-    std::string apparent_mass_balance_side_name = "apparent_mass_balance_" + basalSideName;
-    std::string apparent_mass_balance_RMS_side_name = "apparent_mass_balance_RMS_" + basalSideName;
-    std::string stiffening_factor_side_name = stiffening_factor_name + "_" + basalSideName;
-    std::string effective_pressure_side_name = effective_pressure_name + "_" + basalSideName;
-    std::string vertically_averaged_velocity_side_name = vertically_averaged_velocity_name + "_" + basalSideName;
-    std::string bed_roughness_side_name = "bed_roughness_" + basalSideName;
+    auto bname = [&] (const std::string& fname) -> std::string {
```

Comment:
Yeah, sname and bname are probably not very verbose. I will make them more verbose. And I can turn bname (renamed) into a method, since derived classses might use it too. And add a `surf_fname` method too.